mne/viz/backends/renderer.py

```diff
@@ -52,86 +59,102 @@ def set_3d_backend(backend_name, verbose=None):
+       +--------------------------------------+--------+-----------+----------+
+       | :func:`plot_source_estimates`        | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | :func:`plot_alignment`               | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | :func:`plot_sparse_source_estimates` | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | :func:`plot_evoked_field`            | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | :func:`plot_sensors_connectivity`    | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | :func:`snapshot_brain_montage`       | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | :func:`link_brains`                  |        | ✓         |          |
+       +--------------------------------------+--------+-----------+----------+
+       +--------------------------------------+--------+-----------+----------+
+       | **Feature:**                                                         |
+       +--------------------------------------+--------+-----------+----------+
+       | Large data                           | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | Opacity/transparency                 | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | Support geometric glyph              | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | Smooth shading                       | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | Subplotting                          | ✓      | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | Inline plot in Jupyter Notebook      | ✓      |           | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | Inline plot in JupyterLab            | ✓      |           | ✓        |
+       +--------------------------------------+--------+-----------+----------+
+       | Inline plot in Google Colab          |        |           |          |
+       +--------------------------------------+--------+-----------+----------+
+       | Toolbar                              |        | ✓         | ✓        |
+       +--------------------------------------+--------+-----------+----------+
     """
     global MNE_3D_BACKEND
-    try:
-        MNE_3D_BACKEND
```

Comment:
I don't understand this `try` clause, `MNE_3D_BACKEND=None` is set at the top of the module so it should always exist unless someone does `del mne.viz.backends.renderer.MNE_3D_BACKEND` or so which I don't think we need to support?

Reply:
it looks like it was just used in mocking in tests, I deleted it in favor of something simpler (monkeypatch.setattr)